src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -121,6 +121,9 @@ const CommentView = View.extend({
   regions: {
     comment: '[data-comment-activity-region]',
   },
+  modelEvents: {
+    'change:message': 'render',
+  },
```

Comment:
Pretty sure doing a `render` of the individual comment in question is the correct way... because the comment may be in a "editing" state.